docker/Dockerfile.BuildJekyll

```
@@ -0,0 +1,21 @@
+FROM ruby@sha256:c74394d0a4a05a3068ca9d0c53c36d451e27773264b568ae16ae24be9ad5621d
+# ruby:3.0.0
```

No shade if you did intend it this way, but if you're using buildkit (happy to help you get that updated otherwise) this should be:

```suggestion
FROM ruby:3.0.0@sha256:c74394d0a4a05a3068ca9d0c53c36d451e27773264b568ae16ae24be9ad5621d
```

---

yeah we never had a buildkit discussion did we. I think I don't want to worry about it for now, this was on purpose given that the syntax you're suggestion doesn't currently work. But let's def chat buildkit soon!